codeclimate-test-reporter.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = ">= 1.9"
 
-  spec.add_dependency "simplecov", ">= 0.7.1", "< 1.0.0"
+  spec.add_dependency "simplecov", ">= 0.7.1", "< 0.11.2"
```

Comment:
Should this be `< 0.12.0`? Looking at the github issue - it seems like that's where the problem was